src/components/DropdownMenu.tsx

```
@@ -0,0 +1,75 @@
+import { Menu, MenuButton, MenuItem, MenuItems } from '@headlessui/react';
+import { ChevronDownIcon } from '@heroicons/react/20/solid';
+import React, { useState } from 'react';
+
+export const DropdownMenu: React.FC = () => {
+	const [selectedCourse, setSelectedCourse] = useState<string | null>(null);
+
+	const handleSelect = (course: string) => {
+		setSelectedCourse(course);
+	};
+
+	return (
+		<div className='relative inline-block text-left'>
+			<Menu as='div'>
+				<div>location</div>
+				<div>
+					<MenuButton className='inline-flex w-full justify-center gap-x-1.5 rounded-md bg-white px-8 py-3 text-sm font-semibold text-gray-900 shadow-sm ring-1 ring-inset ring-gray-300 hover:bg-gray-50'>
+						{selectedCourse || 'Select a Golf Course'}
```

Great!, keeps the selected course visible and falls back to the placeholder when nothing is selected.